Automattic/vip-go-mu-plugins
# Search: Add get-last-indexed-post-id command to check the last object ID indexed

## Description
During bulk indexing, we should store the last object ID indexed, especially for very large indexing operations. This also adds a command to check the last object ID indexed.

## Changelog Description

### Plugin Updated: Enterprise Search

Store last object ID processed and add `wp vip-search get-last-indexed-post-id` command to check it.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Scenario A: Incomplete/unsuccessful indexing operation
1. Start indexing operation on a very large site: `wp vip-search index`
2. Stop expectedly after a few have been processed - ctrl + c
3. Use `wp vip-search get-last-indexed-post-id` command to return the last object ID indexed - should match at the point where it was stopped in step 2

Scenario B: Successful indexing operation
1. Start an indexing operation to completion `wp vip-search index`
2. Use `wp vip-search get-last-indexed-post-id` command to return the last object ID indexed - should return a message indicating that there is no last indexed ID has been found
